tests/drum/test_other_cases.py

```diff
@@ -0,0 +1,301 @@
+import os
```

Comment:
file name is too general, please categorize properly.

Reply:
The file is exactly for the cases I can not categorize

Reply:
`test_model_settings` or `test_custom_integration perhaps`?

Reply:
there are many cases

Reply:
test_bin_models_with_wrong_labels
test_detect_language
test_set_language
test_custom_model_with_all_predict_hooks
test_template_creation


They are more like unit tests... but I'm not ready to clean them now.

Reply:
I sitll don't like the name of this file...

Reply:
test_common_cases